src/mono/browser/runtime/marshal.ts

```diff
@@ -42,6 +42,17 @@ export function is_args_exception(args: JSMarshalerArguments): boolean {
     return exceptionType !== MarshalerType.None;
 }
 
+export function is_receiver_should_free(args: JSMarshalerArguments): boolean {
+    if (WasmEnableThreads) return false;
+    mono_assert(args, "Null args");
+    return getB32(<any>args + 20);
```

Comment:
I'm fine with this being a raw constant, but there should be a comment at `internal bool ReceiverShouldFree;` in the definition of `JSMarshalerArgument` to indicate that if the struct's layout changes, we need to update these raw 20s in typescript. Ideally the constant would have a name. We could potentially expand the existing jiterp-consumed API for getting member offsets to return offsets like this too, so they would have named enum values.